example_package/in/abc0a.in

```diff
@@ -1,3 +1 @@
--1 1
-3
-3 2 1
+1 2
```

Comment:
Chyba jednak warto dać jakiś multi-line przykład, żeby było widać jak to się zachowuje

Reply:
można, ale teraz nie mam weny